### DISCUSSION TITLE
Gateway? Proxy? and the like.

### DISCUSSION THREAD

**Discussion by dct-infra** [iptv-org/community#203] [Q&A] [answered]
Hi everyone,

First, forgive my ignorance.
I'm trying to pipe youtube live streams to my Android TV.
Youtube urls work fine on my VLC but not on the TV.

I've been making my own m3u files and using free services like linkastream, and query-streamlink.herokuapp.com (thanks ppl).

Can anyone tell me what is the software used by these services or is it pure dev work? I plan to set up this kind of service on some idle server.

Thanks.

**Comment by Dum4G** [comment, marked as answer]
We use Streamlink and you can learn more about it here
https://github.com/iptv-org/iptv/discussions/2940

**Comment by dct-infra** [comment]
Found what I was looking for, thanks everyone.
